core/arch/arm/plat-hikey/conf.mk

```diff
@@ -21,6 +21,7 @@ CFG_PL011 ?= y
 CFG_HWSUPP_MEM_PERM_PXN ?= y
 CFG_WITH_STACK_CANARIES ?= y
 CFG_TEE_CORE_EMBED_INTERNAL_TESTS ?= n
+CFG_WITH_STATS ?= n
```

Comment:
This is not needed since it will be disabled by default (`mk/config.mk` does not enable it). Same for `CFG_TEE_CORE_EMBED_INTERNAL_TESTS` by the way, which we should probably remove in a separate patch.